src/orquestra/sdk/_ray/_build_workflow.py

```diff
@@ -385,9 +385,12 @@ def _import_pip_env(
             exceptions.OrquestraSDKVersionMismatchWarning,
         )
 
-    return pip_list + [
-        "orquestra-sdk" + (f"=={current_sdk_version}" if current_sdk_version else "")
-    ]
+    # Don't add sdk dependency if submitting from a prerelease or dev version.
+    parsed_sdk_version = version.parse(current_sdk_version)
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**thought:** ‏We have something else that wraps this, but it is not suitable. I think this is OK for now.